libraries/AC_PID/AC_PID_2D.h

```diff
@@ -8,14 +8,15 @@
 #include <stdlib.h>
 #include <cmath>
 #include <AC_PID/AP_PIDInfo.h>
+#include <Filter/SlewLimiter2D.h>
 
 /// @class	AC_PID_2D
 /// @brief	Copter PID control class
 class AC_PID_2D {
 public:
 
     // Constructor for PID
-    AC_PID_2D(float initial_kP, float initial_kI, float initial_kD, float initial_kFF, float initial_imax, float initial_filt_hz, float initial_filt_d_hz);
+    AC_PID_2D(float initial_kP, float initial_kI, float initial_kD, float initial_kFF, float initial_imax, float initial_filt_hz, float initial_filt_d_hz, float initial_srmax=0);
```

Comment:
better to change callers to explicitly pass zero

Reply:
I've remove the SRate parameter completely.